redis/redis/gateway/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM kaazing/enterprise-gateway:latest
+
+# Generate Keystore entry for kaazing.example.com
+
+RUN keytool -genkeypair -keystore conf/keystore.db -storetype JCEKS -keypass ab987c -storepass ab987c -alias kaazing.example.com -keyalg RSA -dname "CN=kaazing.example.com, OU=Example, O=Example, L=Mountain View, ST=California, C=US"
+
+# Add and Build Javascript Client tutorials to web directory so we can test it from a browser,
+# TODO pull in built tutorials when they are released
+
```

Comment:
This TODO can be resolved now that the tutorials are build and can be pulled directly from https://github.com/kaazing/javascript.client.tutorials/releases.  This removes the need to install node and should simplify this